tajo-common/src/main/proto/errors.proto

```diff
@@ -0,0 +1,327 @@
+  SYNTAX_ERROR                          = 602; // SQLState: 42601
+  INSUFFICIENT_PRIVILEGE                = 603; // SQLState: 42501
+
+  UNDEFINED_TABLESPACE                  = 500; // ?
+  UNDEFINED_DATABASE                    = 501; // ?
+  UNDEFINED_SCHEMA                      = 502; // ?
+  UNDEFINED_TABLE                       = 503; // ?
+  UNDEFINED_COLUMN                      = 504; // SQLState: 42703
+  UNDEFINED_FUNCTION                    = 505; // SQLState: 42883
+  UNDEFINED_INDEX                       = 506; // ?
+  UNDEFINED_INDEX_NAME                  = 507; // ?
+  UNDEFINED_PARTITION                   = 508; // ?
+  UNDEFINED_PARTITION_METHOD            = 509; // ?
+  UNDEFINED_OPERATOR                    = 510; // SQLState: 42883 (=UNDEFINED_FUNCTION)
+
+  DUPLICATE_TABLESPACE                  = 510;
+  DUPLICATE_DATABASE                    = 511; // SQLState: 42P04
+  DUPLICATE_SCHEMA                      = 512; // SQLState: 42P06
+  DUPLICATE_TABLE                       = 513; // SQLState: 42P07
+  DUPLICATE_COLUMN                      = 514; // SQLState: 42701
+  DUPLICATE_ALIAS                       = 515; // SQLState: 42712
+  DUPLICATE_FUNCTION                    = 516; // SQLState: 42723
+  DUPLICATE_INDEX                       = 517; // SQLState: ?
+  DUPLICATE_PARTITION                   = 518; // SQLState: ?
+
+  AMBIGUOUS_TABLE                       = 521; // ?
+  AMBIGUOUS_COLUMN                      = 522; // SQLState: 42702;
+  AMBIGUOUS_FUNCTION                    = 523; // SQLState: 42725;
+
+  CANNOT_CAST                           = 604; // SQLState: 42846 - Cast from source type to target type is not supported.
+  GROUPING_ERROR                        = 605; // SQLState: 42803
+  WINDOWING_ERROR                       = 606; // SQLState: 42P20 - PgSQL implementation-defined
+  INVALID_RECURSION                     = 607; // SQLState: 42P19 - PgSQL implementation-defined
+  SET_OPERATION_SCHEMA_MISMATCH         = 608; // SQLState: 42601 (=SYNTAX_ERROR)
+  SET_OPERATION_DATATYPE_MISMATCH       = 609; // SQLState: 42601 (=SYNTAX_ERROR)
+  INVALID_FOREIGN_KEY                   = 621; // SQLState: 42830
+  INVALID_NAME                          = 622; // SQLState: 42602
+  INVALID_COLUMN_DEFINITION             = 631; // SQLState: 42611
+  NAME_TOO_LONG                         = 623; // SQLState: 42622
+  RESERVED_NAME                         = 624; // SQLState: 42939
+  DATATYPE_MISMATCH                     = 625; // SQLState: 42804
+  INDETERMINATE_DATATYPE                = 626; // SQLState: 42P18 - PgSQL implementation -defined
+
+
+
+  // Expressions
+  INVALID_EXPRESSION                    = 701;
+  INVALID_CAST                          = 702;
+  INVALID_DATATYPE                      = 703;
+
+  //NUMERIC_OVERFLOW                    = 803;  // Numeric value overflow
+  //VALUE_LARGER_THAN_PRECISION         = 804;  // Value larger than column precision
```

Comment:
What is the meaning of these commented out codes? If they are TODOs, please add some comments.